build.go

```diff
@@ -133,22 +144,6 @@ func main() {
 	}
 }
 
-func packageGrafana() {
```

Comment:
Unsure what this actually did do and why you removed it :smile:

Reply:
Looks like it set platformArg and pkArch then called createLinuxPackages, pkgArch is now set on line 68 if not defined, and platform arg is now added on line 402 it seems.

Reply:
Yeah, I moved most of the logic out of the function to be able to support sensible defaults. That resulted in the function being too small to be needed.